DependencyInjection/Compiler/ConfigTypesPass.php

```diff
@@ -21,9 +21,18 @@ class ConfigTypesPass implements CompilerPassInterface
     public function process(ContainerBuilder $container)
     {
         $config = $container->getParameter('overblog_graphql_types.config');
-        $generatedClasses = $container->get('overblog_graphql.cache_compiler')->compile($this->processConfig($config));
+        $generatedClasses = $container->get('overblog_graphql.cache_compiler')->compile(
```

Comment:
I think it's a good idea to extract `$container->get('overblog_graphql.cache_compiler')` and check its type just for safety and the IDE will also type hint it nicely